Dockerfile

```diff
@@ -33,8 +33,12 @@ RUN mkdir tessdata && cd tessdata \
     && find . -name '*.traineddata' -maxdepth 2 -exec cp {} /usr/share/tessdata \; \
     && cd .. && rm -r tessdata
 
-COPY dangerzone.py /usr/local/bin/
-RUN chmod +x /usr/local/bin/dangerzone.py
+COPY dangerzone/conversion/common.py /usr/local/bin/
+COPY dangerzone/conversion/pixels_to_pdf.py /usr/local/bin/
+COPY dangerzone/conversion/doc_to_pixels.py /usr/local/bin/
```

Comment:
At the end of the day, the command that runs is:

```
/usr/bin/python3 /usr/local/bin/pixels_to_pdf.py
```

Not that important, but I'd prefer not polluting the `/usr/local/bin` directory, and use a different one (e.g., `/opt/dangerzone`).

Reply:
Things were already in `/usr/local/bin`. We can change it here, sure, but it may be out of scope of the PR. But it's pretty minor if it slips in. However you prefer.

Reply:
Anyways, this is pretty minor. Slipped it in https://github.com/freedomofpress/dangerzone/pull/437/commits/402cc87e8866a27fd13f6b5a68bbf39ae5e23468.

Reply:
You're right, we always had code in `/usr/local/bin`, but now we want to have something like a Python package there, which introduces problems like this: https://github.com/freedomofpress/dangerzone/pull/437#discussion_r1226470694

So, these are new issues that I'd like to nip them in the bud. I'll try to tackle this before we merge this PR, so let's keep this discussion open.